gslib/tests/test_lifecycle.py

```diff
@@ -90,23 +99,34 @@ def test_default_lifecycle(self):
     bucket_uri = self.CreateBucket()
     stdout = self.RunGsUtil(
         ['lifecycle', 'get', suri(bucket_uri)], return_stdout=True)
-    self.assertIn(self.no_lifecycle_config, stdout)
+    if self._use_gcloud_storage:
+      self.assertIn(self.null_lifecycle_config, stdout)
+    else:
+      self.assertIn(self.no_lifecycle_config, stdout)
 
   def test_set_empty_lifecycle1(self):
     bucket_uri = self.CreateBucket()
     fpath = self.CreateTempFile(contents=self.empty_doc1.encode('ascii'))
     self.RunGsUtil(['lifecycle', 'set', fpath, suri(bucket_uri)])
     stdout = self.RunGsUtil(
         ['lifecycle', 'get', suri(bucket_uri)], return_stdout=True)
-    self.assertIn(self.no_lifecycle_config, stdout)
+    if self._use_gcloud_storage:
+      self.assertIn(self.null_lifecycle_config, stdout)
+    else:
+      self.assertIn(self.no_lifecycle_config, stdout)
 
   def test_valid_lifecycle(self):
     bucket_uri = self.CreateBucket()
     fpath = self.CreateTempFile(contents=self.lifecycle_doc.encode('ascii'))
     self.RunGsUtil(['lifecycle', 'set', fpath, suri(bucket_uri)])
     stdout = self.RunGsUtil(
         ['lifecycle', 'get', suri(bucket_uri)], return_stdout=True)
-    self.assertEqual(json.loads(stdout), self.lifecycle_json_obj)
+
+    if self._use_gcloud_storage:
```

Comment:
I think we should match the gsutil output here, else customers might have to modify their existing JSON.

Reply:
Why would people have to modify existing JSON?

Reply:
If I understand correctly based off of the PR, gcloud storage's output will be in the form `{'Lifecylce': {k:v}}`, whereas gsutil's output will be in the form `{k:v}`. This means that if we have to keep the input format same for both gcloud storage and gsutil, the users will have to modify the output from gcloud storage (i.e from the shim) and make it look like the one returned by pure gsutil command? I might be missing something here but this is what I got from the PR.

Reply:
Alternatively, if we are accepting the format `{'Lifecylce': {k:v}}` in gcloud storage, the existing gsutil format might not work.

Reply:
Both gcloud and gsutil accept `{'Lifecycle': {k:v}}` or `{k:v}`

Might be able to remove the `Lifecycle` outer key with a more complex format string using `multi`, but don't see a need given the outer key is accepted by both CLIs.

Reply:
I see. That good that both CLIs accept both format. Even so, I would say, let's just keep the same format using the shim. I would not want to break any other scripts that the users might have that rely on the format outputted by gsutil. Ideally for all get commands, we shouldn't have any modification in the test code accept for stderr messages.

Reply:
Done